docs/lab/02/02.md

```diff
@@ -119,6 +279,13 @@ and vendor dependent, this crate allows developers to get started a little faste
 initialization code and provides a macro called `entry` that developers can use to select the startup (main)
 function.
 
+:::info
+
+Embassy-rs provides the `embassy_executor::main` macro that
+sets the `main` function as the startup point, by also using the `entry` macro, and defines the correct bootloader.
```

Comment:
I would say something like `The entry macro is used by embassy ...`.

Reply:
done